csv_generator/Dockerfile

```diff
@@ -0,0 +1,26 @@
+FROM python:3.6-stretch
+
+# pre-requisites
+RUN apt-get update && \
+    pip install -U pipenv
+
+ARG PROJECT_HOME=/usr/local/csv-generator
+ARG PROJECT_USER=csv-generator
+WORKDIR ${PROJECT_HOME}
+
+RUN useradd -s /bin/bash -d ${PROJECT_HOME} ${PROJECT_USER}
+
+COPY Pipfile .
+COPY Pipfile.lock .
+RUN echo ""
+RUN pipenv install --system --deploy
+
+COPY . ./
+
+RUN chown -R ${PROJECT_USER}: .
+RUN mkdir -p /xml-data && chown -R ${PROJECT_USER}: /xml-data
+RUN mkdir -p /csv-data && chown -R ${PROJECT_USER}: /csv-data
+
+USER ${PROJECT_USER}
+
+ENTRYPOINT ["docker/entrypoint.sh"]
```

Comment:
not completely clear to me, but what was the decision behind the choice between `ENTRYPOINT` and `CMD` in this case?

Reply:
because the script path doesn't have to be specified when using `run`?

Reply:
In the current version it is required to pass parameters. I understand CMD is the default but if I pass parameters to the run command it would replace the command? i.e. I would have to add `python process_xml_zip.py <other paramerers`.

Reply:
Yes, good use case for entrypoint then